spec/helpers/nav_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+# Specs in this file have access to a helper object that includes
+# the NavHelper. For example:
+#
+# describe NavHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+describe NavHelper do
+  describe '#nav_menu_collapsed?' do
+    it "returns true when the nav is collapsed in the cookie" do
+      helper.request.cookies[:collapsed_nav] = "true"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.